ansible/roles/grafana/templates/prometheus.yaml.j2

```diff
@@ -4,6 +4,9 @@ datasources:
 - name: Prometheus
   type: prometheus
   access: proxy
+  basicAuth: true
+  basicAuthPassword: "{{ prometheus_grafana_password }}"
```

Comment:
It's not ideal adding a new password to an existing release - on first deployment this is going to change passwords.yml and require being committed to the kayobe config. It could easily be missed, then regenerated by someone else.

Reply:
We should definitely merge this as-is into Z+A, but perhaps for Yoga the basic auth part should be opt-in?

Reply:
Antelope: https://github.com/stackhpc/kolla-ansible/pull/476
Zed: https://github.com/stackhpc/kolla-ansible/pull/477

Reply:
making basic auth opt in makes this feature pretty complicated to backport and it is diverging from upstream.
I'm not seeing much value added backporting it to yoga in that form - especially that we're in the upgrade phase to antelope anyway